plenum/test/txn_author_agreement/conftest.py

```diff
@@ -27,3 +33,23 @@ def config_req_handler(config_state,
     return ConfigReqHandler(config_ledger,
                             config_state,
                             domain_state=FakeSomething())
+
+
+@pytest.fixture
+def taa_in_data():
+    return [
+        TaaData(randomString(8), randomString(32), n, n + 10)
+        for n in range(10)
+    ]
```

Comment:
I think `taa_input_data` would be less ambiguous name for this fixture. Also using named parameters here would lead to more readable code (`TaaData(version=randomString(8), ...`).

Reply:
updated